plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryMetadata.java

```diff
@@ -376,6 +379,14 @@ private static boolean useStorageApi(ConnectorSession session, String tableName,
         return true;
     }
 
+    private static boolean isBigLakeTable(TableDefinition tableDefinition)
+    {
+        if (tableDefinition instanceof ExternalTableDefinition externalTableDefinition) {
+            return !isNullOrEmpty(externalTableDefinition.getConnectionId()) && externalTableDefinition.getObjectMetadata() == null;
```

Comment:
Can we add a comment on why we are testing for null check ? Can we use `isNullOrEmpty` here ?